components/ImageR/imageR.css

```diff
@@ -0,0 +1,21 @@
+.square-image {
+    height: 9rem;
+    width: 9rem;
+    margin: 10px;
+    object-fit: cover;
```

Comment:
This line is repeated multiple times so you can pass this as a property to your image element in the reset.css file which you might have made for this Library.

Reply:
okay, i'll do that